_layouts/tutorial.html

```diff
@@ -69,6 +69,31 @@ <h3 class="title">Code example:</h3>
           Scala
           </option>
           {% endif %}
+          {% if site.data.tutorials[page.static_data].status.python == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/python.html" {% if page.stack == 'python' %} selected {% endif %}>
+          Python
+          </option>
+          {% endif %}
+          {% if site.data.tutorials[page.static_data].status.csharp == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/csharp.html" {% if page.stack == 'csharp' %} selected {% endif %}>
+          C-sharp/.NET
+          </option>
+          {% endif %}
+          {% if site.data.tutorials[page.static_data].status.go == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/go.html" {% if page.stack == 'go' %} selected {% endif %}>
+          Go
+          </option>
+          {% endif %}
+          {% if site.data.tutorials[page.static_data].status.c == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/c.html" {% if page.stack == 'c' %} selected {% endif %}>
+          C
+          </option>
+          {% endif %}
```

Comment:
A minor point, but I don't see `C` showing up on the page.  But with [librdkafka](https://github.com/edenhill/librdkafka) I'm wondering if we want this option too.  But I'll leave it up to you

Reply:
Great idea @bbejeck .  Added `C (librdkafka)` -- consistent with https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/c.html#client-examples-c

WDYT?